docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,44 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element has to be within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element may be outside of a browser window- as long as TestCafe can reach it by scrolling.
+
+* an element has to be visible - its properties should be as follows:
+
+    Property | Value
+    -------- | --------
+    `display`  | *not* set to `none`
+    `visibility` | set to `visible` (the default value)
+    `width`    | auto calculated as or set to > `0`
+    `height`   | auto calculated as or set to > `0`
+
+* a targeted part of the element has to be visible.  
+
+    TestCafe actions target a center of an element, or a point specified by the `offsetX` and `offsetY` options of an action. If another element obstructs the target point, the action still executes. The [.click](../../reference/test-api/testcontroller/click.md) action, for example, affects an overlapping element instead of a targeted one.
+
+### Example: Scroll an Element into View
+
+TestCafe scrolls to reach items, that are on the page but not on-screen, Therefore, there is no designated scroll action.
```

Comment:
I guess we don't need a comma before `that` (we never need, in contrast to `which`) and there is a typo in `, Therefore`

Reply:
`there is` and `there are` can also be replaced with more specific wording in tech docs:

*TestCafe scrolls to reach items that are on the page but not on-screen. Therefore, TestCafe API does not have a designated scroll action.*

Dirk also usually changes `has no` to `doesn't have`, but it's British vs American